src/bindings/cpp/include/keyset.hpp

```diff
@@ -31,7 +31,9 @@ class KeySet
 	inline KeySet(ckdb::KeySet *k);
 	inline KeySet(const KeySet &other);
 
-	inline explicit KeySet(size_t alloc, va_list ap);
+#if defined(__amd64__) || defined(__x86_64__) || defined(__i386__)
```

Comment:
See 55ae0b7ce62019048c323a06ad3bfc2809f80871

I also expect that it will be used rarely, but it should work properly when needed.